pkgs/os-specific/linux/esdm/default.nix

```
@@ -0,0 +1,81 @@
+{ lib
+, stdenv
+, fetchzip
+, protobufc
+, pkg-config
+, fuse3
+, meson
+, ninja
+, libselinux
+, jitterentropy
+  # A more detailed explaination of the following meson build options can be found
+  # in the source code of esdm.
+  # A brief explanation is given:
+, selinux ? false # meson build option for selinux support
+, drng_hash_drbg ? true  # meson build option for setting the default drng callback
+, drng_chacha20 ? false # meson build option for setting the default drng callback
+, ais2031 ? true # meson build option for setting the seeding strategy to be compliant with AIS 20/31
+, linux-devfiles ? true # meson build option to enable linux /dev/random and /dev/urandom support
+, linux-getrandom ? true # meson build option to enable linux getrandom support
+, es_jitterRng ? true # meson build option to enable support for the entropy source: jitter rng
+, es_cpu ? true # meson build option to enable support for the entropy source: cpu-based entropy
```

"meson build option" is a little redundant.

```suggestion
, es_cpu ? true # enable support for the entropy source: cpu-based entropy
```

---

The comments are now shortened down a bit to be more concise.